### ISSUE TITLE
[BUG[ Fixing the GitHub Pull Request template

### ISSUE THREAD

**Issue by mabruzzo** [enzo-project/enzo-e#364]
The new template for Pull Requests doesn't seem to work right.

When you go to create a new pull-request, the following text automatically populates the description box with the following:

```
---
name: Pull request
about: Create a pull request to help us improve
title: ''
labels: ''
assignees: ''

---

Thank you so much for your PR! To help us review, fill out the form to the best of your ability.  Please make use of the development guide at https://enzo-e.readthedocs.io/en/main/devel/index.html
```

For conciseness, I have clipped the remainder of the message. 

The main point is that the text between the lines reading `---` should probably not appear. 
- As I understand it, this information is a "header" to configure GitHub's behavior.
- The issue-templates have similar "headers" like this. But when you go to open a new issue the "header" information doesn't appear (I think it works properly in that case)

**Comment by mabruzzo** [comment]
@jwise77

**Comment by mabruzzo** [comment]
Resolved by PR #366
